pylace/src/lib.rs

```diff
@@ -1179,6 +1179,71 @@ impl CoreEngine {
         Ok(())
     }
 
+    /// Remove Rows at the given indices.
```

Comment:
I think there could be more information given here. Specifically, let the user know that a DataFrame with the rows removed is returned, AND the Engine is altered

Reply:
Yeah, I was being lazy. Fixed.